pandas/_libs/hashtable_class_helper.pxi.in

```diff
@@ -255,10 +255,56 @@ dtypes = [('Float64', 'float64', 'val != val', True),
           ('UInt64', 'uint64', 'False', False),
           ('Int64', 'int64', 'val == iNaT', False)]
 
+def get_dispatch(dtypes):
```

Comment:
can you add a comment similiar to in algos_take for using the memory view.

Reply:
Done. I also noticed that the indenting was a tad off so I re-indented it to be 4 spaces.